docs/guide/reviewing.md

```diff
@@ -1,16 +1,138 @@
 # Reviewing
 
+## Vocabulary
+
+### Publication
+
+Publications are organizations that manage submissions. Examples of publications can be publishers or newsletters.
+
+![Green circular icon of a book with a bookmark titled, 'Publication'](/public/icons/publication.svg)
+
+### Submission
+
+A work to be collaboritively reviewed. A submission is commonly in a written format. Submissions are uploaded to Pilcrow and submitted by a submitter to a publication.​ Examples of submissions are articles or essays.
+
+![Blue circular icon of a wavy sheet of paper titled, 'Submission'](/public/icons/submission.svg)
+
+### Review
+
+An examination of a submission. Editors, review coordinators, reviewers, and submitters can participate in a review. Reviews are managed by review coordinators.
```

Comment:
```suggestion
An examination of a [submission](./#submission). [Editors](./#editors), [review coordinators](./#review-coordinators), [reviewers](./#reviewers), and [submitters](./#submitters) can participate in a review. Reviews are managed by review coordinators.
```

This might be overkill, I'm not looking to die on the hill of this suggestion ;-)